wrappers/javascript/aries-askar-nodejs/tests/keys.test.ts

```diff
@@ -1,10 +1,19 @@
 import { Key, KeyAlgs, KeyMethod } from '@hyperledger/aries-askar-shared'
 
+import { ariesAskarNodeJS } from '../src'
+
 describe('keys', () => {
   beforeAll(() => {
     require('@hyperledger/aries-askar-nodejs')
   })
 
+  test('supported backends', () => {
+    const backends = ariesAskarNodeJS.keyGetSupportedBackends()
+
+    expect(backends.length).toStrictEqual(1)
+    expect(backends).toStrictEqual(expect.arrayContaining(['software']))
+  })
```

Comment:
Maybe it should also support returning which backends are supported for which key, but I'm als fine to add that in the future

Reply:
Yeah that would be a good one. A bit more difficult to check dynamically, but it should be fine to track that statically.